homework.js

```diff
@@ -0,0 +1,119 @@
+// 1
+function numberSum(n) {

```

Comment:
the function name should follow the read me and be named summation

Reply:
Also "n" is a bad variable name, please use something longer and represents the variable.